### DISCUSSION TITLE
HX Need Calibration 的参数问题

### DISCUSSION THREAD

**Discussion by tommy9112** [206cc/PicoBETH#8] [General]
![image](https://github.com/user-attachments/assets/edbe9d5b-0d1f-4c5f-9c20-ad002b9c61dc)
旧版是这样的。数值是15000
新版变成这样，请问原理是什么呢？因为我的总是通不过。所以改大了请教下。
![image](https://github.com/user-attachments/assets/44a9baa4-3aa8-44fa-92bf-201167a9b9df)

**Comment by 206cc** [comment]
需要張力校正提醒，是不是校正完沒有再點旁邊的S儲存? 重開機之後變回舊的校正參數造成?

**Reply by tommy9112** [reply]
是的，阅读后才发现要S储存。然后发现之前拉的拍有一半的没有校准过的XD
其实这个应该自动保存就可以了。

**Reply by 206cc** [reply]
最早是自動儲存的，後來我發現我家小朋友沒事很喜歡亂按玩這台機器，導致校正跑掉我也不知道，就加了一個儲存的步驟，其實這個校正選項應該藏更深的，我的可靠性測試機跑了50幾萬次，用張力器測準度完全一樣不需校正，基本上只要沒換 loadcell、hx711機板、沒有微調珠夾斜度就終身不用校正。

**Reply by tommy9112** [reply]
是的，之前不要存储，拆拆装装发现参数几乎不怎么变。所以我都不知道新的要存储，一直自己找是不是机器或者是程序的问题。
